.github/workflows/scripts/sensitive_file_check.py

```
@@ -0,0 +1,232 @@
+        config_file: Path to the configuration file containing regex patterns.
+
+    Returns:
+        result: List of regex patterns.
+
+    """
+    if not os.path.exists(config_file):
+        print(f"Error: Configuration file '{config_file}' not found.")
+        sys.exit(1)
+
+    result = []
+    with open(config_file, "r", encoding="utf-8") as fh_:
```

Ruff (0.15.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)